src/app/store.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import homeReducer from '../features/home/homeSlice';
+
+const rootReducer = combineReducers({
+  home: homeReducer,
+});
+
+/* eslint-disable-next-line import/prefer-default-export */
```

Comment:
- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉